_modules/virl_core.py

```diff
@@ -27,7 +27,7 @@
 os_username = 'admin'
 password = 'password'
 OS_TENANT = 'admin'
-OS_AUTH_URL = 'http://localhost:5000/v2.0'
```

Comment:
Wait a second, I don't see you reading it from env in your tree.   Only reverting.